src/main/java/racingcar/Car.java

```diff
@@ -11,6 +17,19 @@ public int moveForward(int value){
         return currentPosition;
     }
 
+    public String getName() {
+        return name.name();
+    }
+    public int getPosition() {
+        return currentPosition;
+    }
+
+    public int maxPosition(int lastPosition){
+        if(currentPosition > lastPosition)
+            return currentPosition;
+        return lastPosition;
```

Comment:
조금 더 간단하게 변경해 보면 아래처럼 가능하겠네요😁
```suggestion
        return currentPosition > lastPosition ? currentPosition : lastPosition;
```
더 간단히 하면 요렇게요😁
```suggestion
        return Math.max(currentPosition, lastPosition);
```